### ISSUE TITLE
new feature request: all search in specific ontology and all ontology at same time

### ISSUE THREAD

**Issue by nicolevasilevsky** [EBISPOT/OLS#454]
If I am on a page for a specific ontology term, there is  search box for search [specific ontology], which sometimes I want to do, but sometimes I want to search all ontologies, in which case, I need to go back to the home page.

It would be super cool if there was a way to do both from the same page without having to go back.

If I was on a page such as [this](https://www.ebi.ac.uk/ols/ontologies/doid/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FDOID_5426), could there be two search bars?

![image](https://user-images.githubusercontent.com/6722114/105893590-209a1980-5fc8-11eb-9bf6-6845d04c2677.png)

or maybe a toggle button?
![image](https://user-images.githubusercontent.com/6722114/105893792-5fc86a80-5fc8-11eb-901e-2c593c8450b2.png)

**Comment by cmungall** [comment]
yeh I often end up having to do the back button thing

I feel two search boxes are awkward. I like Having one, but having a clear filter that is set with an "x" next to it that can remove the filter

**Comment by nicolevasilevsky** [comment]
I love the idea of a removable filter, @cmungall, good idea.

**Comment by henrietteharmse** [comment]
Hi Nicole & Chris,

Thank you for your suggestions. We will certainly consider this.

I have a question. Why not have 2 tabs open for OLS? One for searching all ontologies and one for searching a specific ontology. I find this useful because I can have the 2 instances open side by side.

**Comment by nicolevasilevsky** [comment]
Good point, that would be a workaround. It would still be ideal to have the ability to search on a specific and all ontologies on the same page though. :) I like the idea of turning on/off a filter that Chris suggested.

**Comment by jseager7** [comment]
I also support this, as I usually expect the search box to search all ontologies rather than the one I'm currently browsing.

Another suggestion is to have two search buttons, one which retains the default behaviour of searching the current ontology, and another that searches all ontologies:

![image](https://user-images.githubusercontent.com/37659591/106147035-4f9ebf80-616f-11eb-9ea8-ff7ddfe08576.png)

You could have these as different colours to help users distinguish them:

![image](https://user-images.githubusercontent.com/37659591/106146960-35fd7800-616f-11eb-85bd-7d03a002b5cc.png)

That means users wouldn't have to keep toggling the filter if the setting isn't remembered between visits or across different ontology pages.

**Comment by nicolevasilevsky** [comment]
Fantastic idea @jseager7!
